assets/responsibleai/vision/components/rai_vision_insights/spec.yaml

```diff
@@ -116,7 +116,7 @@ outputs:
     type: path
     description: Json file to which UX is serialized to for viewing in static AzureML Studio UI
 code: ../src
-environment: azureml://registries/azureml/environments/responsibleai-vision/versions/1
+environment: azureml://registries/azureml/environments/responsibleai-vision/versions/2
```

Comment:
not completely sure if we want to do this yet, given automl is still on older version of pytorch

Reply:
revert to /responsibleai-vision/versions/1